src/app/support-page/support-page.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-support-page',
+  templateUrl: './support-page.component.pug',
+  styleUrls: ['./support-page.component.scss']
+})
+export class SupportPageComponent implements OnInit {
```

Comment:
We may be able to get rid of the constructor and OnInit if we don't implement OnInit 

```suggestion
export class SupportPageComponent {
```